src/diffusers/pipelines/pixart_alpha/pipeline_pixart_alpha.py

```diff
@@ -625,6 +670,7 @@ def __call__(
         prompt: Union[str, List[str]] = None,
         negative_prompt: str = "",
         num_inference_steps: int = 20,
+        original_inference_steps: int = None,
```

Comment:
Let's add this to the docstrings too.

Reply:
Do we really need to add this to the `__call__` method? I think one can always set this once for the scheduler no?:

```py
pipe.scheduler.register_to_config(original_inference_steps=...)
```